modules/planning/tasks/optimizers/path_time_heuristic/st_graph_point.cc

```diff
@@ -71,5 +71,11 @@ void StGraphPoint::SetPrePoint(const StGraphPoint& pre_point) {
   pre_point_ = &pre_point;
 }
 
+const double StGraphPoint::GetOptimalSpeed() const { return optimal_speed_; }
```

Comment:
No need to return a const copy, just a copy.

Reply:
Fixed. Thank you!